terraform/environments/digital-prison-reporting/application_variables.json

```diff
@@ -241,11 +241,11 @@
       "compact_raw_job_log_level": "INFO",
       "compact_raw_job_schedule": "cron(45 1 * * ? *)",
       "compact_structured_job_worker_type": "G.1X",
-      "compact_structured_job_num_workers": 2,
+      "compact_structured_job_num_workers": 3,
```

Comment:
This adds more workers to prevent the compaction job from timing out when the data size is large. Typically happens after the offender_course_attendances has been loaded.